modules/auxiliary/scanner/mssql/mssql_hashdump.rb

```diff
@@ -25,14 +25,14 @@ def initialize
 
   def run_host(ip)
 
-    if !mssql_login_datastore
+    if !mssql_login(datastore['USERNAME'], datastore['PASSWORD'])
       print_error("Invalid SQL Server credentials")
       return
     end
 
     service_data = {
         address: ip,
-        port: rport,
+        port: datastore['RPORT'],
```

Comment:
Why do we need this change? 👀

Reply:
Per offline conversation, leaving this as is pending further testing

Reply:
I believe these can all be reverted too, as the mssql mixin still brings in the remote tcp mixin:

```
include Exploit::Remote::Tcp
```